### ISSUE TITLE
Chart is expanding endless

### ISSUE THREAD

**Issue by cyromat** [highcharts/highcharts#21188]
#### Expected behaviour
The chart should be created and stay as it is

#### Actual behaviour
The chart gets created but then gets bigger and bigger.
It never stops.

#### Live demo with steps to reproduce
[] (https://jsfiddle.net/p2tckvzy/1/)

div {
	padding: 2px;
}
is the problem.

#### Product version
v11.4.1

v11.4.0 works fine

#### Affected browser(s)
Chrome, FF, Edge

**Comment by karolkolodziej** [comment]
Hi @cyromat! 
Thank you for reporting the issue! 

Internal note:
Bisected to: https://github.com/highcharts/highcharts/commit/cd8ffc1dcada95e3cce67be0ac6d364229a6b577
FYI @hubertkozik

**Comment by oysteinmoseng** [comment]
Internal note: There are some hidden divs that should be visually hidden when including the accessibility module. Maybe those are not forced hidden enough (padding 0, width/height: 1px etc). Also figure out why it's bisected to that thing. Possible that this could happen with even older versions?

**Comment by hubertkozik** [comment]
Bisected to aaf02f563e923e3e85fd8bba6938f88d314db79b. Previous bisect was wrong.

**Comment by hubertkozik** [comment]
The workaround is to set a specific height to the chart's container div.

**Live demo:** https://jsfiddle.net/BlackLabel/kweL4xhv/

`#container {
    height: 400px;
}`
